src/raft/uv_fs.c

```diff
@@ -704,10 +704,18 @@ static int probeDirectIO(int fd, size_t *size, char *errmsg)
 			default:
 				/* UNTESTED: this is an unsupported file system.
 				 */
-#if defined(__s390x__)
+#if defined(__alpha__)
+				ErrMsgPrintf(errmsg,
+					     "unsupported file system: %x",
+					     fs_info.f_type);
+#elif defined(__s390x__)
 				ErrMsgPrintf(errmsg,
 					     "unsupported file system: %ux",
 					     fs_info.f_type);
+#elif defined(__x86_64__) && defined(__ILP32__)
+				ErrMsgPrintf(errmsg,
+					     "unsupported file system: %llx",
+					     fs_info.f_type);
 #else
```

Comment:
From `statfs(2)`:

> The `__fsword_t` type used for various fields in the `statfs` structure definition is a glibc internal type, not intended for public use.  This leaves the programmer in a bit of a conundrum when trying to copy or compare these fields to local variables in a program.  Using `unsigned int` for such variables suffices on most systems.

Casting to an `unsigned int` or `unsigned long` should work; I copied the ifdefs since there was already one there for s390x.

Reply:
Thanks, I hadn't noticed the existing ifdef -- we should remove that too! (No need to do it in this PR.) Casting to `unsigned long` would be my preferred approach here.